### PR TITLE
Added a travis file

### PR DESCRIPTION
**Pull request by etorreborre**
This should help to validate all the future PRs coming from @filippovitale :-) (cf #1).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: scala
+
+scala:
+  - 2.11.5
+
+jdk:
+  - oraclejdk7
+
+script: sbt -sbt-version 0.13.7 -scala-version $TRAVIS_SCALA_VERSION 'test-only -- xonly'
```
Comment (charleso) [inline .travis.yml]: What happens with the default?

**Comment by charleso** [comment]
:+1:

File: .travis.yml (hunk 1 above)
Comment (etorreborre) [inline .travis.yml]: No idea TBH, I got this from another config.

**Comment by filippovitale** [comment]
:construction_worker: :+1:

**Pull request merged**